#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *
from math import pi

PACKAGE = "laser_filters"

gen = ParameterGenerator()

##### gen.add("name", paramtype, level, description, default, min, max)

gen.add("lower_angle", double_t, 0, "Minimum angle before cutoff", -1, -pi, pi)
gen.add("upper_angle", double_t, 0, "Maximum angle before cutoff", 1, -pi, pi)
gen.add("switch_", bool_t, 0, "Switches filter on or off", True)

exit(gen.generate(PACKAGE, "laser_filters", "AngularBoundsFilterInPlace"))